app/components/entity_mentions/integration.py

```diff
@@ -3,43 +3,27 @@
 import discord
 
 from .fmt import entity_message
-from app.utils import MessageLinker, is_dm, is_mod, remove_view_after_timeout, try_dm
+from app.utils import (
+    DeleteMessage,
+    MessageLinker,
+    is_dm,
+    remove_view_after_timeout,
+    try_dm,
+)
 
 IGNORED_MESSAGE_TYPES = frozenset(
     (discord.MessageType.thread_created, discord.MessageType.channel_name_change)
 )
 
+WARN_MSG_TEMPLATE = "Only the person who mentioned {} can remove this message."
```

Comment:
I feel like this same message template is going to be used everywhere, so perhaps moving it to utils or hardcoding it in the `DeleteMessage` class itself would be desirable.  It would definitely serve to increase the consistency of warning messages throughout the bot.

Of course, the verb would change:
```suggestion
WARN_MSG_TEMPLATE = "Only the person who {verb} {noun} can remove this message."
```
(I'm not particularly fond of `noun` but I couldn't think  of anything better right now; feel free to use whatever you want here, this is just a suggestion.)

Reply:
How about `action_singular` and `action_plural` that would just be e.g. `"mentioned this entity"` and `"mentioned these entities"`?

Reply:
Actually I think it would look better to simply have a single placeholder:
```suggestion
WARN_MSG_TEMPLATE = "Only the person who {} can remove this message."
```

Reply:
That's what I'm suggesting ^^

Reply:
Oh, GitHub didn't load your message so when I sent that it looked like I had two subsequent comments 😆.

Reply:
> How about `action_singular` and `action_plural` that would just be e.g. `"mentioned this entity"` and `"mentioned these entities"`? The template would be part of the base class.

Fine by me.